src/elements/common/item/iconCellRenderer.js

```diff
@@ -1,32 +1,15 @@
 /**
  * @flow
- * @file Function to render the icon table cell
```

Comment:
Do we need to add a different description for the file now?

Reply:
@benjamin-shen remember to address this if needed

Reply:
Ooh thanks, good catch!! I added it back here: https://github.com/box/box-ui-elements/pull/3238, could you approve @JChan106 ?